templates/file_viewer_template.html

```diff
@@ -54,6 +54,12 @@
                 // Change either the selected button or option in dropdown
                 // ani_state.current_image
 
+                // Guard clause to prevent run time exception
+                if (imgs.length === 0) {
+                    // Exit callback early to short-circuit TypeError
+                    return
+                }
+
```

Comment:
@claretandy this is probably not a great idea since it'll hinder future debugging sessions by swallowing the bug. I think I'll revert it and submit a new PR with a `errorHandler` or something to inform the user of the site that there is an incompatible menu setting